hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsMesosConstraints.java

```diff
@@ -0,0 +1,104 @@
+package org.apache.mesos.hdfs.scheduler;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apache.mesos.Protos.Attribute;
+import org.apache.mesos.Protos.Offer;
+import org.apache.mesos.Protos.Value.Range;
+
+/**
+ * HDFS Mesos offer constraints checker class implementation.
+ */
+public class HdfsMesosConstraints {
+
+  private final Map<String, String> constraints;
+
+  public HdfsMesosConstraints(Map<String, String> mesosSlaveConstraints) {
+    this.constraints = mesosSlaveConstraints;
+  }
```

Comment:
This is how it works. However, inside catch I am bound to put some statement because of style checks. found=false is unnecessary, but just added to avoid style check. I can't log an error there, because it not error or warning..